src/controller/model/endpoint.ts

```diff
@@ -628,9 +628,18 @@ class Endpoint extends Entity {
             this.save();
         } catch (error) {
             const err = error as Error;
-            err.message = `${log} failed (${err.message})`;
-            logger.debug(err.stack!, NS);
-            throw error;
+            if ((error as Zdo.StatusError).code == 136) {
```

Comment:
```suggestion
            if ((error as Zdo.StatusError).code == Status.NO_ENTRY) {
```

(pick it from the `zdo/definition/status.ts`)